.gitignore

```diff
@@ -1,5 +1,12 @@
-.sizecache.json
-bower_components/
-external
-node_modules
-tmp/
+doc
+spec/spec/db/*
+vendor
+NOTES
+.DS_Store
+.bundle
+Gemfile.lock
+.rvmrc
+*.sw*
+*.gem
+.rbx
+.ruby-version
```

Comment:
These files are not included by Globalize, therefore they shouldn't be listed here.